src/docs/cookbook-payments/removing-stripe/index.md

```diff
@@ -301,7 +301,7 @@ Stripe related components:
 You can also remove Stripe script form
 [public/index.html](https://github.com/sharetribe/ftw-daily/blob/master/public/index.html#L169).
 
-# Removing payments
+### Removing payments
```

Comment:
A question: Is the idea to change this from h1 to h3? Or should this be a h2?

Reply:
You are right, this should be a h2